src/coreclr/src/vm/multicorejit.cpp

```diff
@@ -548,6 +675,32 @@ void MulticoreJitRecorder::RecordJitInfo(unsigned module, unsigned method)
     }
 }
 
+void MulticoreJitRecorder::RecordNDirectStubInfo(unsigned module, unsigned method)
```

Comment:
Do we really need a dedicated recording path for PInvoke methods? Can they be recorded as regular methods and only special cased during replay?

Reply:
It is a good idea. I will update. Thank you.